src/components/dialogs/connectivity/connectivity-form.jsx

```diff
@@ -165,14 +170,34 @@ export const ConnectivityForm = ({
             forcePopupIcon
             name={`${id}.${VOLTAGE_LEVEL}`}
             label={voltageLevelSelectLabel}
+            previousValue={
+                isEquipmentModification ? previousValues?.voltageLevelId : null
+            }
             options={voltageLevelOptions}
             getOptionLabel={getObjectId}
             size={'small'}
         />
     );
 
-    const connectedField = (
-        <SwitchInput name={`${id}.${CONNECTED}`} label="connected" />
+    const previousConnected = useMemo(() => {
+        if (previousValues?.terminalConnected) {
+            return intl.formatMessage({ id: 'On' });
```

Comment:
Not sure for 'On' and 'Off'. To be checked with PO. Could be 'Connected'/'Disconnected'